apache/zeppelin
# [ZEPPELIN-869] Search notebook to handle empty result.

### What is this PR for?
Search notebook to handle empty result.

### What type of PR is it?
Bug Fix
### Todos


### What is the Jira issue?
[ZEPPELIN-869](https://issues.apache.org/jira/browse/ZEPPELIN-869?jql=project%20%3D%20ZEPPELIN%20AND%20text%20~%20%22empty%22)

### How should this be tested?
1. Try to search something which should return empty result, you should get proper message for that.
2. search box should get cleared when clicked on any other link.



### Screenshots (if appropriate)
Before:
![screen shot 2016-06-09 at 3 44 53 pm](https://cloud.githubusercontent.com/assets/7026661/15926338/25f8924e-2e59-11e6-8c10-2dd9542f5258.png)

After:
![screen shot 2016-06-09 at 3 34 47 pm](https://cloud.githubusercontent.com/assets/7026661/15926303/f1412d7c-2e58-11e6-9dd3-1ecccb938a26.png)
### Questions:
* Does the licenses files need update?No
* Is there breaking changes for older versions?No
* Does this needs documentation?No
